.github/workflows/ci.yaml

```diff
@@ -4,142 +4,159 @@ on:
   #workflow_dispatch: #manual start from Github UI or API
   push:
     branches: master
-  #pull_request:
-    #branches: master
+  pull_request:
+    branches: master
 
 jobs:
-  main_job:
-    runs-on: ubuntu-latest
-    permissions:      
-      contents: read  #permissions for publishing Docker images to Github Packages
-      packages: write #permissions for publishing Docker images to Github Packages
-          
+  test:
+    timeout-minutes: 30
+    runs-on: ubuntu-20.04
     defaults:
       run:
-        working-directory: angular 
+        working-directory: angular
+
+    outputs:
+      start_time: ${{ steps.start.outputs.start_time}}
 
     strategy:
       matrix:
         node-version: [16.x]
-       #node-version: [16.x, 14.x] #used this line instead of the one above to emulate errors in the test phase AND to test matrix syntax
+
     steps:
       - uses: actions/checkout@v3
+
+      - name: start time
+        id: start
+        run: |
+          printf -v now '%(%s)T'
```

Comment:
this is a little bit cryptic. Why not to use 
```
➜ date  +%s
1661219076

```

so you get a proper value in seconds, which then you can easily operate on?